.github/workflows/lint.yaml

```diff
@@ -0,0 +1,119 @@
+name: lint
+on: [push, pull_request]
+env:
+  GH_ACTION: "TRUE"
```

Comment:
`GITHUB_ACTIONS` is implicitly always set to `true` by default, so you could use that instead, see https://docs.github.com/en/actions/learn-github-actions/variables#default-environment-variables.

Reply:
Exactly what I'd come to say!

Although I'd personally probably favour problem matchers over making our scripts GitHub Actions specific, they're pretty easy to do, even if you might need to tweak our script output to be easier to parse:
https://github.com/actions/toolkit/blob/main/docs/problem-matchers.md

Reply:
Changed the env var! See comment below about moving to problem matchers later (something is better than nothing for now).

Reply:
Implemented the problem matchers after all!